jumpscale/servers/threebot/threebot.py

```diff
@@ -199,15 +199,18 @@ class Package:
     def __init__(self, path, default_domain, default_email, giturl=""):
         self.path = path
         self.giturl = giturl
-        self.config = self.load_config()
-        self.name = self.config["name"]
+        self._config = None
+        self.name = j.sals.fs.path_parts(f"{path}")[-1]
```

Comment:
why not j.sal.fs.basename? and if it's the same variable and no need to format it should be `j.sals.fs.path_parts(path)`

Reply:
I used ```j.sals.fs.path_parts(path)``` because if we add  ```/ ``` 
 in the end of the path like 
``` /sandbox/code/github/threefoldtech/js-sdk/jumpscale/packages/marketplace/ ```path_parts function igonre it and return basename but basename function will return / not basename.